### PR TITLE
Top bar integration

### PR DESCRIPTION
**Pull request by SinemSungur**
Hi! i build top bar side of the main page. I just confused about how to remove the default color of anchor tag. Could you show me the way. Cus i search it but it did not work on my project. Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,195 +4,460 @@
...
-              >
-            </li>
-            <li class="top-container-item">
-              <a href="#"><i class="fas fa-download"></i> DOWNLOAD APP</a>
-            </li>
+            </div>
+            <div class="col-sm-12 col-md-9 p-lg-0 text-lg-right">
+              <ul class="top-link-container">
+                <li class="top-link-item d-md-none d-lg-block">
+                  <a href="tel:+1800-123-7890"
```
Comment (ofcyln) [inline index.html:45]: DO you mean :hover state?
If yes:
Screenshot 2020-06-29 at 21 10 17

Comment (ofcyln) [reply index.html:45]: You can inspect the element and select the hover checked. then you can see the hover color over the inspector.

Comment (SinemSungur) [reply index.html:45]: ok i solved the problem :)

File: index.html [hunk 2]
```diff
@@ -4,195 +4,460 @@
...
-              <a href="#">SELL/RENT YOUR PROPERTY</a>
-              <p>
-                Genuine Clients, 100% Trust Assurance and Lowest Fees on the
-                Market.
-                <a href="#"> Learn More <i class="fas fa-arrow-right"></i> </a>
-              </p>
-            </li>
-            <li class="top-container-item">LOGIN/REGISTER</li>
-          </ul>
+                <li class="top-link-item">
```
Comment (ofcyln) [inline index.html:55]: When I hover over the 'SELL/RENT YOUR PROPERTY' a tag, I can't go on opened element. There is one challenge to fix this problem. I'll give you time for solving it. This fix will make you understand the parent/child relationship and the power of the CSS usage at the end.
cant-click-on-hover

File: style.css [hunk 3]
```diff
@@ -82,27 +82,31 @@ ul.top-item-submenu {
...
   font-weight: 400;
   text-transform: capitalize;
   border-top: 1px solid #2c3e50;
-  z-index: 1000;
-  transition: 500ms all;
-  right: 0;
-}
+  margin: 0;
+} 
 .top-bar-dropdown2 {
```
Comment (ofcyln) [inline style.css:96]: Container is this element and the children needs to be under this parent classes. Hover logic should be integrated under this!

**Pull request merged**